### PR TITLE
FIX issues recently raised during a global review

### PR DESCRIPTION
**Pull request by fgalan**
Some little fixes:

* Example in multitenancy section
* Example regarding how TimeInstant is used
* Example for subscriptions and registrations

### CODE REVIEW

File: docs/topics/subcriptions_and_registrations.md [hunk 1]
```diff
@@ -19,6 +44,34 @@ On the one hand, subscriptions:
 
 On the other hand, registrations:
 
+    POST /v1/registry/registerContext
```
Comment (frbattid) [inline docs/topics/subcriptions_and_registrations.md]: Are registrations also available for NGSIv2? If that's the case, I think it is better using v2.

Comment (fgalan) [inline docs/topics/subcriptions_and_registrations.md]: I'm afraid not yet :(

NTC

Comment (frbattid) [inline docs/topics/subcriptions_and_registrations.md]: OK, so then it is OK.

**Comment by frbattid** [comment]
LGTM

**Pull request merged**